AnanasProject/AnanasProject/ViewControllers/SearchViewController/WKWebViewController.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  WKWebViewController.swift
+//  AnanasProject
+//
+//  Created by Анастасия Козлова on 11.10.2022.
+//
+
+import UIKit
+import WebKit
```

Comment:
```suggestion
import WebKit

```
ставь пробелы после импортов всегда!